article/app/controllers/ArticleController.scala

```diff
@@ -161,7 +161,7 @@ object ArticleController extends Controller with RendersItemResponse with Loggin
 
   def renderArticle(path: String) = {
     LongCacheAction { implicit request =>
-      mapModel(path) {
+      mapModel(path, request.isEmail) {
```

Comment:
I'd put the variable name of the boolean value here:
`mapModel(path, blocks = request.isEmail)`